health_check_storage/base.py

```diff
@@ -20,7 +21,7 @@ class MyStorageHealthCheck(StorageHealthCheck):
     storage = None
 
     def get_storage(self):
-        if isinstance(self.storage, basestring):
+        if isinstance(self.storage, text_type):
```

Comment:
I did not want to add an additional dependency and six shipped with django, hence the import location.